src/main/java/sirius/biz/storage/layer2/variants/ConversionProcess.java

```diff
@@ -110,9 +114,57 @@ public void upload(Runnable task) {
      * Used by the {@link Converter} to supply the actual result of the conversion.
      *
      * @param fileHandle the file to use as variant
+     * @return the object itself for fluent method calls
      */
-    public void withConversionResult(FileHandle fileHandle) {
+    public ConversionProcess withConversionResult(FileHandle fileHandle) {
         this.fileHandle = fileHandle;
+        return this;
+    }
+
+    /**
+     * Provides the file to use as input for the conversion.
+     * <p>
+     * This is useful when the file already exist in the file system, skipping a new download from storage.
+     *
+     * @param fileToConvert the file to use as input
+     * @return the object itself for fluent method calls
+     * @see #obtainInputFile()
+     */
+    public ConversionProcess withInputFile(File fileToConvert) {
+        this.fileToConvert = fileToConvert;
+        return this;
+    }
+
+    /**
+     * Returns a file handle to an input file for conversion.
+     * <p>
+     * This can either be a permanent file handle over a previously supplied file or a temporary one over a fresh
+     * downloaded file from the {@linkplain #blobToConvert blob}.
```

Comment:
?

```suggestion
     * This can either be a permanent file handle pointing to a previously supplied file, or a temporary one
     * pointing to a file freshly downloaded from the {@linkplain #blobToConvert blob}.
```